.eslintrc.js

```diff
@@ -5,4 +5,12 @@ module.exports = {
     'eslint-comments/no-unlimited-disable': 0,
     'eslint-comments/no-unused-disable': 0,
   },
+  overrides: [
+    {
+      files: ['*.js'],
+      rules: {
+        '@typescript-eslint/explicit-function-return-type': 0,
+      },
+    },
+  ],
```

Comment:
Lets handle this in https://github.com/dooboolab/eslint-config/commit/3eb4f36511da28e0f3733cef3ea1bf8b3fbed572

Reply:
I'll handle this!